examples/inverse/plot_dics_source_power.py

```diff
@@ -32,9 +34,10 @@
 ###############################################################################
 # Reading the raw data and creating epochs:
 data_path = somato.data_path()
-raw_fname = data_path + '/MEG/somato/sef_raw_sss.fif'
-fname_fwd = data_path + '/MEG/somato/somato-meg-oct-6-fwd.fif'
-subjects_dir = data_path + '/subjects'
+subject = '01'
+task = 'somato'
+raw_fname = op.join(data_path, 'sub-{}'.format(subject), 'meg',
+                    'sub-{}_task-{}_meg.fif'.format(subject, task))
```

Comment:
I know I'm late on the review cycle and this is merged, but do you guys find worth to create a helper function that allows composing the name?

```py
def fname_helper(subject, task='somato', data_type_prefix=''):
    return op.join(
        'sub-{0:02d}'.format(subject),
        data_type_prefix,
        'sub-{0:02d}_task-{}_meg.fif'.format(subject, task)
    )
```

```py
from mne.datasets.somato import data_path, fname_helper

raw_fname = op.join(data_path(), fname_helper(subject=1, data_type_prefix='meg'))
fname_fwd = op.join(data_path, 'derivatives', fname_helper(subject=1, data_type_prefix='meg'))
```

Reply:
ups !! the file names are not the same. 

one is `_meg.fif` the other one is `_fwd.fif`

Reply:
The derivatives is not set in stone, so I would not make a new function. There is already one in mne-bids for the existing BIDS spec: https://mne.tools/mne-bids/generated/mne_bids.make_bids_basename.html#mne_bids.make_bids_basename.